2020-05-20 01:09:38.816 ERROR 8320 --- [XNIO-1 I/O-4] lavalink.server.io.SocketContext         : Error

java.nio.channels.ClosedChannelException: null
	at io.undertow.server.protocol.framed.AbstractFramedStreamSinkChannel.flush(AbstractFramedStreamSinkChannel.java:347) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
	at org.xnio.ChannelListeners$14.handleEvent(ChannelListeners.java:413) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
	at org.xnio.ChannelListeners$14.handleEvent(ChannelListeners.java:409) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92) ~[xnio-api-3.3.8.Final.jar!/:3.3.8.Final]
	at io.undertow.server.protocol.framed.AbstractFramedStreamSinkChannel$1.run(AbstractFramedStreamSinkChannel.java:214) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
	at io.undertow.server.protocol.framed.AbstractFramedChannel$1.run(AbstractFramedChannel.java:137) ~[undertow-core-2.0.26.Final.jar!/:2.0.26.Final]
	at org.xnio.nio.WorkerThread.safeRun(WorkerThread.java:582) ~[xnio-nio-3.3.8.Final.jar!/:3.3.8.Final]
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:466) ~[xnio-nio-3.3.8.Final.jar!/:3.3.8.Final]

2020-05-20 01:09:43.815  INFO 8320 --- [XNIO-1 I/O-4] lavalink.server.io.SocketServer          : Connection closed from /127.0.0.1:56124 with status CloseStatus[code=1006, reason=] -- Session can be resumed within the next 60 seconds with key 0.s1kze6ay5nm
2020-05-20 01:10:43.816  INFO 8320 --- [pool-13-thread-1] lavalink.server.io.SocketContext         : Shutting down 0 playing players.
2020-05-20 01:15:16.924  WARN 8320 --- [lava-daemon-pool-manager-3-thread-1] c.s.d.l.tools.GarbageCollectionMonitor   : Suspicious GC results for the last 2 minutes: [Bucket 0 = 0] [Bucket 20 = 0] [Bucket 50 = 0] [Bucket 200 = 1] [Bucket 500 = 0] [Bucket 2000 = 0] 
